Formula/h/hdf5.rb

```diff
@@ -33,6 +33,12 @@ class Hdf5 < Formula
 
   conflicts_with "hdf5-mpi", because: "hdf5-mpi is a variant of hdf5, one can only use one or the other"
 
+  # upstream commit to fix c++ regression: https://github.com/HDFGroup/hdf5/pull/4473
```

Comment:
Let's make this more actionable 
```suggestion
  # fix c++ regression, can be removed in next release 
```

Reply:
ok